Wikipedia/Code/DiffListUneditedCell.swift

```diff
@@ -5,22 +5,19 @@ class DiffListUneditedCell: UICollectionViewCell {
     static let reuseIdentifier = "DiffListUneditedCell"
     
     @IBOutlet var innerLeadingConstraint: NSLayoutConstraint!
-    @IBOutlet var innerTrailingConstraint: NSLayoutConstraint!
     @IBOutlet var innerTopConstraint: NSLayoutConstraint!
     @IBOutlet var innerBottomConstraint: NSLayoutConstraint!
     @IBOutlet var textLabel: UILabel!
     @IBOutlet var divView: UIView!
-    @IBOutlet var textBackgroundView: UIView!
     
     func update(_ viewModel: DiffListUneditedViewModel) {
         innerLeadingConstraint.constant = viewModel.innerPadding.leading
-        innerTrailingConstraint.constant = viewModel.innerPadding.trailing
         innerTopConstraint.constant = viewModel.innerPadding.top
         innerBottomConstraint.constant = viewModel.innerPadding.bottom
         
         textLabel.font = viewModel.font
-        textLabel.text = viewModel.text
-        
+        textLabel.text = viewModel.text.capitalized(with: Locale.current)
```

Comment:
An alternative here is just `.localizedCapitalized` which we also use elsewhere.